FirebaseInAppMessaging/Sources/DefaultUI/FIRIAMRenderingWindowHelper.m

```diff
@@ -50,7 +54,11 @@ + (UIWindow *)UIWindowForBannerView {
 #if defined(__IPHONE_13_0) && __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000
     if (@available(iOS 13.0, *)) {
       UIWindowScene *foregroundedScene = [[self class] foregroundedScene];
-      UIWindowForBanner = [[FIRIAMBannerViewUIWindow alloc] initWithWindowScene:foregroundedScene];
+      if (foregroundedScene.delegate) {
+        UIWindowForBanner = [[UIWindow alloc] initWithWindowScene:foregroundedScene];
```

Comment:
Should this be `FIRIAMBannerViewUIWindow` instead of `UIWindow`?

Even better would be to refactor to get rid of the duplicated code.

Reply:
Fixed both in 14c31bb.